specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-02-21-preview/Subscription.json

```diff
@@ -0,0 +1,214 @@
+                  }
+               }
+            },
+            "parameters":[  
+               {  
+                  "$ref":"#/parameters/apiVersionParameter"
+               }
+            ]
+         }
+      },
+      "/{scope}/providers/Microsoft.Subscription/createSubscription":{  
+         "post":{  
+            "description":"Creates an Azure subscription",
+            "operationId":"createAzureSubscription",
+            "x-ms-long-running-operation": true,
+            "responses":{  
+               "200":{  
+                  "description":"Normal response for a successful query. The response body will contain the data that matches the filters specified in the query parameters.",
+                  "schema":{  
+                     "$ref":"#/definitions/SubscriptionCreationResult"
+                  }
+               },
+               "202":{  
+                  "description":"Accepted. Subscription creation is in progress.",
+                  "headers":{  
+                     "Location":{  
+                        "description":"GET this URL to retrieve the status of the asynchronous operation.",
+                        "type":"string"
+                     },
+                     "Retry-After":{  
+                        "description":"The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                        "type":"string"
+                     }
+                  }
+               }
+            },
+            "parameters":[  
+               {  
+                  "name":"scope",
+                  "in":"path",
+                  "description":"Uri path corresponding to commercial subscription or legacy subscription",
+                  "required":true,
+                  "type":"string"
+               },
+               {  
+                  "$ref":"#/parameters/apiVersionParameter"
+               },
+               {  
+                  "name":"body",
+                  "in":"body",
+                  "description":"The subscription creation request object.",
+                  "required":true,
+                  "schema":{  
+                     "$ref":"#/definitions/SubscriptionCreationRequest"
+                  }
+               }
+            ]
+         }
+      }
+   },
+   "definitions":{  
+      "SubscriptionCreationResult":{  
+         "properties":{  
+            "subscriptionId":{  
+               "type":"string"
+            }
+         }
+      },
+      "SubscriptionCreationRequest":{  
+         "properties":{  
+            "subscriptionDisplayName":{  
+               "type":"string"
+            },
+            "Owners":{  
+               "type":"array",
+               "items":{  
+                  "$ref":"#/definitions/AdPrincipal"
+               },
+               "description":"An array of Active Directory Principals"
+            },
+            "offerId":{  
+               "type":"string"
```

Comment:
Add: "description": "The offer type of the subscription. For example, MS-AZR-0017P (EnterpriseAgreement) and MS-AZR-0148P (EnterpriseAgreement devTest) are available. Only valid when creating a subscription in a billing account scope."